opennms-web-api/src/main/java/org/opennms/web/svclayer/support/DefaultDistributedStatusService.java

```diff
@@ -248,7 +248,7 @@ public SimpleWebTable createStatusTable(DistributedStatusDetailsCommand command,
                           "bright");
             table.addCell(getResponseText(s.getPollResult()), "");
             table.addCell(reDateify(s.getPollResult().getTimestamp()), "");
-            //TODO: table.addCell(reDateify(s.getLocationMonitor().getLastUpdated()), "");
+            table.addCell(reDateify(new Date()), ""); // TODO Patrick: from where do we get the date?
```

Comment:
@fooker we should discuss this.

Reply:
This can be ignored(stay as you currently have it) for now - the whole code will hopefully go away soon.